### DISCUSSION TITLE
IOS如何使用我们的软件

### DISCUSSION THREAD

**Discussion by YayuZhou123** [getlantern/lantern-forum-cn#653] [General]
你好，苹果手机也想使用我们的软件。怎么操作？ 从哪里能下载 IOS 蓝灯APP?

**Comment by MaxMKMax** [comment]
@YayuZhou123 您好. 您可以在这里下载. 谢谢!
https://lantern.io/zh-Hans/download?os=ios

https://github.com/getlantern/lantern
